### PR TITLE
Docker dev server

### PR DESCRIPTION
**Pull request by KaiVolland**
This modifies the webpack.dev.config.js to work properly with shogun-docker. See updated README.md for details.

It also adds the `Select` and `Markdown` (real markdown editor will be added in a followup) keys to the `GeneralEntityForm`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,23 @@
...
+`shogun-admin` should be used in an Docker environment. The easiest way to do this is to use [shogun-docker](https://github.com/terrestris/shogun-docker) which includes the .
 
-Requirements:
+# Development
 
-- A runnning shogun instance
-- Keycloak-Preparation
-  - The user you use to login needs the role `admin`
-  - The path of the `shogun-admin` needs to be added to `Valid Redirect URIs` and `Web Origins`
+The WebPack dev server is configured to generate output into `dist_dev`.
```
Comment (dnlkoch) [inline README.md:12]:
```suggestion
The webpack dev server is configured to generate output into `dist_dev`.
```

File: README.md [hunk 2]
```diff
@@ -3,21 +3,23 @@
...
+    image: nexus.terrestris.de/repository/terrestris-public/shogun-admin:latest
+    ports:
+      - 8005:80
+    volumes:
+      - ../../shogun-admin/dist_dev:/var/www/html
 ```
-
-The `shogun-admin` expects shogun itself to serve a `client-config.js` file which configures the admin client. If this file is not found, the `fallbackConfig.js` from the assets folder is loaded. You can use this as a starting point for your own `client-config.js`.
+> ⚠️ Currently the dev server will spam your console with websocket warnings as hot reloading is not
+proberly configured for the docker scenario.
```
Comment (dnlkoch) [inline README.md:25]:
```suggestion
properly configured for the docker scenario.
```

File: webpack.dev.config.js [hunk 3]
```diff
@@ -1,77 +1,35 @@
...
-      '/actuator',
-      '/cache',
-      '/sso',
-      '/v2',
-      '/config'
-    ],
-    target: 'https://localhost/'
-  }]
+devConfig.devServer = {
+  webSocketServer: false,
```
Comment (dnlkoch) [inline webpack.dev.config.js:29]: So currently no hot reloading is supported?

**Review by dnlkoch — APPROVED** [review]

File: webpack.dev.config.js (hunk 3 above)
Comment (KaiVolland) [reply webpack.dev.config.js:29]: Correct. You have to refresh the page. I tried several configs in the nginxes and webserver.
We can fixes this in a followup.

**Pull request merged**